src/components/CortexPage/CortexPage.tsx

```diff
@@ -27,6 +27,8 @@ import { cortexApiRef } from '../../api';
 import { Permission } from '../../api/types';
 import { extensionApiRef } from '../../api/ExtensionApi';
 import { HelpPage } from '../HelpPage';
+import {isBeforeShutdownDate, shouldShowExpirationBanner} from "../Entitlements/ExpirationUtils";
+import {ExpirationBanner} from "../Entitlements/ExpirationBanner";
```

Comment:
Eep we clearly don't run prettier checks in CI...
```suggestion
import { isBeforeShutdownDate, shouldShowExpirationBanner } from '../Entitlements/ExpirationUtils';
import { ExpirationBanner } from '../Entitlements/ExpirationBanner';
```